Makefile

```diff
@@ -47,10 +51,6 @@ shell: build
 	$(DOCKER_RUN_TRAEFIK) /bin/bash
 
 image: build
-	if ! [ -a dist/traefik_linux-386 ] ; \
-	then \
-		$(DOCKER_RUN_TRAEFIK) ./script/make.sh generate binary; \
-	fi;
 	docker build -t $(TRAEFIK_IMAGE) .
```

Comment:
I'm making the assumption this is mostly always called in `circleci` and/or when the binary already exists.